src/effects/fireworks/index.ts

```diff
@@ -0,0 +1,167 @@
+/*
+ Copyright 2020 Nurjin Jafar
+ Copyright 2020 Nordeck IT + Consulting GmbH.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import ICanvasEffect from '../ICanvasEffect';
+
+export type FireworksOptions = {
+    /**
+     * the maximum number of the fireworks
+     */
+    maxCount: number,
+    /**
+     * the alpha opacity of the fireworks (between 0 and 1, where 1 is opaque and 0 is invisible)
+     */
+    gravity: number,
```

Comment:
The description is not matching the property. also not all options currently have a description.